### PR TITLE
CSF 3 conversion

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@react-aria/slider/stories/Slider.stories.tsx [hunk 1]
```diff
@@ -1,121 +1,176 @@
...
+  <StorySlider
+    label="Size"
+    onChange={action('onChange')}
+    onChangeEnd={action('onChangeEnd')}
+    showTip />
+);
+
+Single.story = {
+  name: 'single'
+};
```
Comment (snowystinger) [inline packages/@react-aria/slider/stories/Slider.stories.tsx:25]: while you can do it this way, we've been doing it with an object declaration, lets keep with that
```
Single = {
  render...
  name...
}
```

Comment (LFDanLu) [reply packages/@react-aria/slider/stories/Slider.stories.tsx:25]: right, the current code style is from the codemod, figured I'd get everything converted first so I can get the storybook axe + interaction test stuff in and then change the style to match our other stories

**Comment by LFDanLu** [comment]
Running out of time in the tech week here for the rest of the format conversion (aka https://github.com/adobe/react-spectrum/pull/4399#pullrequestreview-1389128295). I'd like to merge this in as is since it is still a comprehensive CSF3 conversion and will unblock the accessibility test work from https://github.com/adobe/react-spectrum/pull/3711.

**Review by snowystinger — APPROVED** [review]
Did a cursory sweep of the stories. I don't see any bad patterns in the transformation that would lead to it breaking.

**Review by reidbarber — APPROVED** [review]
LGTM. I do see some names that got changed to be auto generated, but thats fine. You can do `yarn storybook:build`, then `npx storybook extract {build_dir} {output.json}` and compare diffs in the build output in terms of story information.

**Comment by LFDanLu** [comment]
@reidbarber Thanks for the tip, I was wondering about how to do that myself when I ran into the new name formatting causing Chromatic to think some of the stories were brand new. Will try that

**Comment by LFDanLu** [comment]
Diff of build output for anyone who is curious (first output is from the csf3 conversion build, 2nd is from main): https://www.diffchecker.com/IsLng3qS/. Didn't notice anything too strange, not sure what the changes in the importPath and stuff mean

**Pull request merged**